src/model/time_entry.cc

```diff
@@ -223,10 +223,6 @@ void TimeEntry::SetDescription(const std::string &value, bool userModified) {
         SetDirty();
 }
 
-void TimeEntry::SetStopString(const std::string &value, bool userModified) {
```

Comment:
you forgot to remove it from the `.h` file

Reply:
Done 👍